core/modules/link/link.diff.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * @file
+ * Provide diff field functions for the Link module.
+ */
+
+/**
+ * Diff field callback for parsing link fields comparative values.
+ */
+function link_field_diff_view($items, $context) {
+  $diff_items = array();
+  foreach ($items as $delta => $item) {
+    if ($item['url'] && $item['title']) {
+      $diff_items[$delta] = $item['title'] . ' (' . $item['url'] . ')';
+    }
+    else {
+      $diff_items[$delta] = $item['title'] . $item['url'];
+    }
+  }
+  return $diff_items;
+}
```

Comment:
I removed this.
I don't think there is a `diff` module port yet so no corresponding issue filed.